app/assets/javascripts/controllers/container_dashboard/util/chart-data-mixin.js

```diff
@@ -0,0 +1,6 @@
+angular.module('miq.util').factory('ChartsDataMixin', ['$timeout', '$q', function chartDataMixinFactory ($timeout, $q) {
```

Comment:
no 'use strict' here? 
also consider having 'use strict' as the first line of the file, then it applies for everything